e2e/signout-modal.spec.ts

```diff
@@ -49,7 +50,7 @@ test.describe('Signout Modal component', () => {
     await expect(
       page.getByRole('dialog', { name: translations.signout.heading })
     ).not.toBeVisible();
-    await expect(page).toHaveURL(/.*\/learn\/?$/);
+    await expect(page).toHaveURL(allowTrailingSlash('/learn'));
```

Comment:
These two are not equivalent. Does that matter?

Reply:
Technically, no, but in practice they're the same. The `.*` matches anything, but, since `allowTrailingSlash` doesn't have a leading `^` it can start the match anywhere.

What that means is that if we have a url `http://localhost:8000/espanol/learn/` then the first regex will match  `http://localhost:8000/espanol/learn/` and the second just `/learn/`, but they both match and that's all that matters.